lib/eventq/eventq_aws/sns.rb

```diff
@@ -87,6 +87,10 @@ def find_topic(topic_name)
         end
 
         arn
+      rescue Aws::SNS::Errors::Throttling => error
+        EventQ.logger.error("[#{self.class}] - #find_topic: #{error.message}} - retrying in 2s")
+        sleep 2
+        find_topic(topic_name)
```

Comment:
```suggestion
        retry
```

I would also suggest:

1. A maximum retry count. I know it shouldn't be an issue, but...
2. If we consider this "handled"; update the method's documentation above